charts/velero/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: 1.4.2
+appVersion: 1.5.1
 description: A Helm chart for velero
 name: velero
-version: 2.12.17
+version: 2.12.18
```

Comment:
should be 
```suggestion
version: 2.13.0
```

Reply:
it is a minor change in the app, should be reflected in the chart as well.

There is any breaking change? if a user has the previous version and upgrades to this one, everything still works?

Reply:
If this is the last thing to do, can you please @svcua do this? to let every of us enjoy 1.5 :)